svc-aggregation/go.mod

```diff
@@ -3,12 +3,23 @@ module github.com/ODIM-Project/ODIM/svc-aggregation
 go 1.13
 
 require (
-	github.com/ODIM-Project/ODIM/lib-dmtf v0.0.0-20200727115727-33d557ff397c
-	github.com/ODIM-Project/ODIM/lib-messagebus v0.0.0-20201007090649-06a4c52ce4cf
-	github.com/ODIM-Project/ODIM/lib-persistence-manager v0.0.0-20201021053518-88345d4a9988 // indirect
-	github.com/ODIM-Project/ODIM/lib-rest-client v0.0.0-20201007090649-06a4c52ce4cf
-	github.com/ODIM-Project/ODIM/lib-utilities v0.0.0-20201016064732-c652b5935f93
+	github.com/Joker/hpp v1.0.0 // indirect
+	github.com/ODIM-Project/ODIM/lib-dmtf v0.0.0-20201201072448-9772421f1b55
+	github.com/ODIM-Project/ODIM/lib-messagebus v0.0.0-20201201072448-9772421f1b55
+	github.com/ODIM-Project/ODIM/lib-persistence-manager v0.0.0-20201201072448-9772421f1b55 // indirect
+	github.com/ODIM-Project/ODIM/lib-rest-client v0.0.0-20201201072448-9772421f1b55
+	github.com/ODIM-Project/ODIM/lib-utilities v0.0.0-20201201072448-9772421f1b55
 	github.com/fsnotify/fsnotify v1.4.9
+	github.com/k0kubun/colorstring v0.0.0-20150214042306-9440f1994b88 // indirect
```

Comment:
why this package is added

Reply:
This package got added after the go mod tidy